tests/test_ssm.py

```diff
@@ -47,3 +47,60 @@ def test_ssm_parameter_not_secure(self):
         resources = p.run()
         self.assertEqual(len(resources), 1)
         self.addCleanup(client.delete_parameters, Names=['test-name', 'secure-test-name'])
+
+    @functional
```

Comment:
this isn't a functional test, it depends on manually setup context that is not created as part of the test for the test to run. functional decorator should only be used for fully self-contained tests.